docs/devcontainers/devcontainer-cli.md

```diff
@@ -64,6 +55,8 @@ Commands:
   devcontainer build [path]         Build a dev container image
   devcontainer run-user-commands    Run user commands
   devcontainer read-configuration   Read configuration
+  devcontainer features             Features commands
```

Comment:
We can probably highlight "open" as an option when installed via VS Code.

Reply:
Added a note on this.